contracts/modules/capital/Pool.sol

```diff
@@ -202,6 +204,31 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
     return swapDetails[assetAddress];
   }
 
+  function getAssetsAndSwapDetails() external view returns (
```

Comment:
Should we exclude abandoned assets from being copied over? I could change this function so it only returns active assets.